jax/_src/distributed.py

```diff
@@ -130,6 +145,7 @@ def initialize(coordinator_address: str | None = None,
                num_processes: int | None = None,
                process_id: int | None = None,
                local_device_ids: int | Sequence[int] | None = None,
+               spec_detection_method: str | None = None,
```

Comment:
Nit: can we name this "cluster_detection_method"?

"spec" isn't a term known to the user.

Reply:
Sure, done.